docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -433,15 +475,24 @@ Below is a table of the tags that are currently supported, and their correspondi
 | `cssborder`   | Not supported |   Use `string` as workaround |
 
 
-###### 3.2.2 Migrate Page field to Page (eZ Enterprise only)
+###### 3.2.2 Migrate eZ Matrix
 
-**If** you use Page field (ezflow) and an eZ Enterprise subscription, and are ready to migrate your eZ Publish Flow content to the eZ Enterprise Page format, you can use a script to migrate your old Page content to new Page, to start using a pure eZ Enterprise setup. See [Migrating legacy Page field (ezflow) to new Page (Enterprise)](#migrating-legacy-page-field-ezflow-to-new-page-enterprise) for more information.
+**If** you use Matrix field (ezmatrix), you'll need to [migrate the storage format](../updating/4_update_2.5.md#changes-to-matrix-field-type)
+as it has changed to json internally for new eZ Platform field type.
 
 ###### 3.2.3 Add other eZ Enterprise schemas (eZ Enterprise only)
 
-For date-based publisher and form builder, there are additional tables, you can import them to your database using the following sql files:
-`<new-ez-root>/vendor/ezsystems/date-based-publisher/bundle/Resources/install/datebasedpublisher_scheduled_version.sql`,
-`<new-ez-root>/vendor/ezsystems/ezstudio-form-builder/bundle/Resources/install/form_builder.sql`, `<new-ez-root>/vendor/ezsystems/ezstudio-notifications/bundle/Resources/install/ezstudio-notifications.sql`
+In eZ Platform, the system ships with additional enterprise features that need to be installed.
+
+1. First add all tables needed by eZ Platform Enterprise:
+   - https://github.com/ezsystems/ezplatform-ee-installer/blob/2.4/Resources/sql/schema.sql#L198
```

Comment:
Sure we can start on line 198 and skip everything before?

When I do upgrades, I [remove all the "drop table ..." statements in the file](https://github.com/vidarl/ezp-toolkit/blob/master/database/upgrade_db.sh#L25) and then apply the whole thing. I don't think any tables in that file will exists if upgrading from 5.4

Reply:
```suggestion
   - https://github.com/ezsystems/ezplatform-ee-installer/blob/2.4/Resources/sql/schema.sql
```